py_swf/clients/workflow.py

```diff
@@ -65,3 +66,181 @@ def terminate_workflow(self, workflow_id, reason):
+            oldest_start_date=oldest_start_date,
+            latest_start_date=latest_start_date,
+        )
+
+        return self.boto_client.count_open_workflow_executions(
+            domain=self.workflow_client_config.domain,
+            startTimeFilter=start_time_filter_dict['startTimeFilter'],
+            **workflow_filter_dict
+        )
+
+    def count_closed_workflow_executions(
+            self,
+            oldest_start_date=None,
+            latest_start_date=None,
+            oldest_close_date=None,
+            latest_close_date=None,
+            workflow_name=None,
+            version=None,
+            tag=None,
+            workflow_id=None,
+            close_status=None,
+    ):
+        """
+        Count the number of closed workflows for a domain. You can pass in filtering criteria.
+        This operation is eventually consistent. The results are best effort and may not exactly reflect recent updates
+        and changes. Worklfows started or closed near the time when calling count_open_workflow_executions may not be reflected
+
+        Passthrough to :meth:~SWF.Client.count_closed_workflow_executions``
+
+        startTimeFilter and closeTimeFilter are mutually exclusive. You MUST specify one of these in a request but not both.
+        closeStatusFilter , executionFilter , typeFilter and tagFilter are mutually exclusive. You can specify at most
+            one of these in a request.
+
+        :param oldest_start_date: datetime. Specifies the oldest start date and time to return.
+        :param latest_start_date: datetime. Specifies the latest start date and time to return.
+        :param oldest_close_date: datetime. Specifies the oldest close date and time to return.
+        :param latest_close_date: datetime. Specifies the latest close date and time to return.
+        :param workflow_name: string. Required for typeFilter. Specifies the type of the workflow executions to be counted.
+        :param version: string. Optional for typeFilter. Version of the workflow type.
+        :param tag: string. Required for tagFilter. Specifies the tag that must be associated with the execution for it
+            to meet the filter criteria.
+        :param workflow_id: string. Required for executionFilter. The workflowId to pass of match the criteria of this filter.
+        :param close_status: string.
+            valid status are ('COMPLETED', 'FAILED', 'CANCELED', 'TERMINATED', 'CONTINUED_AS_NEW', 'TIMED_OUT')
+            This filter has an affect only if executionStatus is specified as CLOSED
+        :return: total number of closed workflows that meet the filter criteria
+        """
+        time_filter_dict = _build_time_filter_dict(
+            oldest_start_date=oldest_start_date,
+            latest_start_date=latest_start_date,
+            oldest_close_date=oldest_close_date,
+            latest_close_date=latest_close_date,
+        )
+
+        count_closed_workflow_within_time_range = partial(
```

Comment:
I don't think we can unpack workflow_filter_dict while having the time_filter_dict unpacked as key word dict. Or am I understanding it wrong?